index.php

```diff
@@ -52,11 +82,11 @@
                     option('bvdputte.kirbyAutopublish.webhookToken', false) === false
                 ) {
                     throw new Exception('Invalid token');
-                    return false;
```

Comment:
I removed the `false` statement because your're throwing an Exception here and it is never executed.